editor/src/components/canvas/canvas-component-entry.tsx

```diff
@@ -32,12 +37,19 @@ export const CanvasComponentEntry = betterReactMemo(
         store.dispatch,
       ),
     }))
-    return (
-      <UiJsxCanvas
-        {...canvasProps}
-        clearErrors={props.clearErrors}
-        reportError={props.reportError}
-      />
-    )
+
+    if (canvasProps.uiFilePath == null) {
+      return null
+    } else {
+      return (
+        <CanvasErrorBoundary uiFilePath={canvasProps.uiFilePath} reportError={props.reportError}>
+          <UiJsxCanvas
+            {...canvasProps}
+            clearErrors={props.clearErrors}
```

Comment:
does the canvas still use clearErrors and reportError?

Reply:
Yup.